css/amp.css

```diff
@@ -777,3 +777,49 @@ amp-accordion > section[expanded] > :last-child {
 amp-story-page {
   background-color: #757575;
 }
+
+amp-date,
```

Comment:
I think only `layout="container"` will work since the React component creates the overlay in the DOM as a child of the component.

Reply:
I think this is necessary for the unattached styles, because custom elements are not `display: block;` by default. The AMP layout will override this when it attaches anyway, since it has such low specificity.